example_scenes/basic.py

```diff
@@ -29,7 +29,7 @@ def construct(self):
         transform_title.to_corner(UP + LEFT)
         self.play(
             Transform(title, transform_title),
-            LaggedStart(*map(FadeOutAndShiftDown, basel)),
+            LaggedStart(*map(lambda obj: FadeOutAndShift(obj, direction=DOWN), basel)),
```

Comment:
While we are at it, we could also remove the map here:

```suggestion
            LaggedStart(*[FadeOutAndShift(obj, direction=DOWN) for obj in basel]),
```